### ISSUE TITLE
LOSS meaniou no change

### ISSUE THREAD

**Issue by wangjing60755** [DrSleep/light-weight-refinenet#59]
when I run src/train.py use  mbv2,  the LOSS is 0.001 and meaniou is 0.335544 and no change, why ??

shell ::::

_2020-06-18 21:10:02,115 :: INFO :: __main__ :: Training: stage 0 epoch 0
Loss 0.002 | Avg. Loss 0.002: 100%|?????????????????????????????????????| 4543/4543 [3:02:05<00:00,  2.40s/it]
/home/bigdata/anaconda3/envs/hapi_torch/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:143: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
2020-06-19 00:12:07,290 :: INFO :: __main__ :: Validation: stage 0 epoch 0
meaniou : 0.335544: 100%|?????????????| 1554/1554 [2:12:07<00:00,  5.10s/it]
--------------------
2020-06-19 02:24:14,528 :: INFO :: densetorch.misc.utils ::  New best value 0.3355, was 0.0000
2020-06-19 02:24:14,587 :: INFO :: __main__ :: Training: stage 1 epoch 0
Loss 0.001 | Avg. Loss 0.001: 100%|?????????????????| 4543/4543 [2:59:06<00:00,  2.37s/it]
2020-06-19 05:23:21,063 :: INFO :: __main__ :: Validation: stage 1 epoch 0
meaniou : 0.335544: 100%|????????????????| 1554/1554 [2:07:59<00:00,  4.94s/it]
--------------------
2020-06-19 07:31:21,056 :: INFO :: __main__ :: Training: stage 2 epoch 0
Loss 0.001 | Avg. Loss 0.001:  63%|????????????????            | 2875/4543 [1:49:27<1:02:43,  2.26s/it]_

**Comment by DrSleep** [comment]
what script are you running and on what dataset?

**Comment by wangjing60755** [comment]
> what script are you running and on what dataset?

celebA

**Comment by wangjing60755** [comment]
> what script are you running and on what dataset?

celebA, run src_v2/train.py

**Comment by DrSleep** [comment]
I have not worked with celebA, probably it needs some tuning
